spring-kafka/src/main/java/org/springframework/kafka/support/LoggingProducerListener.java

```diff
@@ -32,7 +32,10 @@
  *
  * @author Marius Bogoevici
  * @author Gary Russell
+ * @deprecated since 3.0 in favor of {@link org.apache.kafka.clients.producer.ProducerInterceptor}
```

Comment:
Well, we don't have a `LoggingProducerInterceptor` to replace with.
I wonder if we need to implement one...